docs/api.md

```diff
@@ -1020,39 +1231,59 @@ Example Response Body:
 {
   "standardActions": [
     {
-      "uuid": "b95fa3fc-84a7-46f1-9ce0-1eca1b144117",
-      "type": "DEPOSIT",
-      "owner": "0x014be43bf2d72a7a151a761a1bd5224f7ad4973c",
-      "transferAmount": "33720746949513441",
-      "price": "6227979999899999000000",
+      "uuid": "f2f0ac19-373f-4a80-bd34-a8d973ee0235",
+      "type": "OPEN",
+      "owner": "0x77a035b677d5a0900e4848ae885103cd49af9633",
+      "number": null,
+      "transferAmount": null,
+      "tradeAmount": "990000",
+      "price": "90.85",
       "market": "PBTC-USDC",
-      "side": "LONG",
-      "orderNumber": "956855500050000",
-      "confirmedAt": "2020-02-27T23:10:31.000Z",
-      "createdAt": "2020-02-27T23:11:31.758Z",
-      "updatedAt": "2020-02-27T23:11:31.778Z",
-      "product": "perpetual",
-    },
+      "asset": null,
+      "side": "SHORT",
+      "operationUuid": null,
+      "transactionHash": "0x7c6367c2058e8c121437181a3e535a1ea06271ecf47be449c0217bdd0c785ad9",
+      "positionUuid": null,
+      "borrowAmount": null,
+      "orderNumber": "1011122300520002",
+      "confirmedAt": "2020-05-21T19:49:10.000Z",
+      "feeAmount": "454250",
+      "feeAsset": "USDC",
+      "pnl": null,
+      "payoutAmount": null,
+      "isPendingBlock": false,
+      "refundAmount": "0",
+      "product": "PERPETUAL",
+      "createdAt": "2020-05-21T19:49:48.039Z",
+      "updatedAt": "2020-05-21T19:49:48.039Z"
+    }
   ]
 }
 ```
 
-### Standard Action Response Object
+#### Standard Action Response Object
 
 | Field Name     | Description                                                   |
 |----------------|---------------------------------------------------------------|
 | uuid           | The unique id for the action.                                 |
 | owner          | The wallet address of the user.                               |
-| type           | The type of standard action e.g. `DEPOSIT`.                   |
+| type           | The type of standard action e.g. `OPEN` or `ACCOUNT_SETTLE`.  |
```

Comment:
Would it be helpful to define what the acceptable types here or anywhere or is this not the place ?

Reply:
I think we could have a type glossary or similar at the end? We could also have separate detailed docs for the different types.

Reply:
added the types for solo and perpetual